app/assets/stylesheets/application.scss

```diff
@@ -31,4 +30,22 @@
  h4, p {
     display: grid;
     padding-left: 5px;
-}
\ No newline at end of file
+}
+
+table {
+    border: 1px solid black;
+    width: 80%;
+    padding: 5px;
+
+    td {
+        border: 1px solid black;
+        padding: 5px;
+    }
+    tr {
```

Comment:
Be mindful of the code formatting, no line in between here but you have a line in between in other places